client/lib/user-agent-utils/index.js

```diff
@@ -0,0 +1,7 @@
+export function currentUserAgent() {
+	return navigator.userAgent;
+}
```

Comment:
this might be a good question for @ehg to weigh in on. we don't gain much right now as far as testability and server-side rendering is concerned by putting this behind a function, but there _might_ be a way to provide that benefit